src-built-in/components/advancedAppLauncher/src/utils/sort-functions.js

```diff
@@ -33,3 +33,11 @@ export default {
 		});
 	}
 }
+
+const isAppInFavorites = (appID) => {
+	const favorites = storeActions.getSingleFolder('Favorites').apps;
+	const index = findIndex(favorites, favorite => favorite.appID === appID);
```

Comment:
Is there a way to make a helper function for all your findindex uses? There are a lot at this point.